File: fac.c
#include <stdio.h>

int main(){
    int number, fac =1;
    
    printf("Enter a factorial number: ");
    scanf("%d", &number);

    for (int i = 1; i <= number; i++){
          fac *= i;
}
    printf("Factorial of %d is %d\n" ,number, fac);
    return 0;

}

File: fac.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Enter a number: ", function(number) {
  number = parseInt(number);
  let fact = 1;
  for (let i = 1; i <= number; i++) {
    fact *= i;
  }
  console.log(`Factorial of ${number} is ${fact}`);
  rl.close();
});
